app/lib/server/lib/processDirectEmail.js

```diff
@@ -1,6 +1,9 @@
 import { Meteor } from 'meteor/meteor';
+import { Random } from 'meteor/random';
 import { EmailReplyParser as reply } from 'emailreplyparser';
 import moment from 'moment';
+import base64 from 'base-64';
+import utf8 from 'utf8';
```

Comment:
Use the native module https://nodejs.org/docs/latest-v10.x/api/string_decoder.html